build-config/Makefile

```diff
@@ -317,7 +322,11 @@ ifeq ($(PARTED_ENABLE),yes)
   include make/parted.make
 endif
 ifeq ($(GRUB_ENABLE),yes)
-  include make/grub.make
+  ifeq ($(ARCH),arm64)
+    include make/grub_aarch64.make
+  else
+    include make/grub.make
+  endif
```

Comment:
This is ugly including a separate arch specific grub.make at this level.  i would put the necessary aarach64 changes in grub.make.  However, I haven't go to the grub changes -- maybe what you have here will make sense after I read that section.